library/src/main/java/com/mypopsy/widget/FloatingSearchView.java

```diff
@@ -621,12 +621,6 @@ private void parseMenu(XmlPullParser parser, AttributeSet attrs)
     }
 
     static private Drawable unwrap(Drawable icon) {
-        if(icon instanceof android.support.v7.graphics.drawable.DrawableWrapper)
```

Comment:
This is the part that was crashing, I'm not sure why the conditions were here, maybe you got some valid reasons, however, some of these classes/methods are not accessible anymore